.github/workflows/main.yaml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
```

Comment:
Lets limit to one platform for now and add other platforms when we will have bug reports of that kind.